jcl/src/openj9.jvm/share/classes/com/ibm/jvm/Dump.java

```diff
@@ -177,13 +177,16 @@ public static void SnapDump() {
     	SnapDumpImpl();
     }
     
-	private static native int SnapDumpImpl();
-    
 	private static final DumpPermission DUMP_PERMISSION = new DumpPermission();
 	private static final ToolDumpPermission TOOL_DUMP_PERMISSION = new ToolDumpPermission();
 	private static final String LEGACY_DUMP_PERMISSION_PROPERTY = "com.ibm.jvm.enableLegacyDumpSecurity"; //$NON-NLS-1$
 	
-    private static final class DumpOptionsLock {}
+	private static final class DumpOptionsLock {
+    	/* empty */
+		DumpOptionsLock() {
+			/* constructor for enclosing class */
+		}
```

Comment:
The comments here are not helpful; please remove the 'empty' comment, replace the comment in the constructor with a call to `super();`.